cashflows/flow.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from .settings import FLOW_TYPES
+from .util import Time
+
+
+class Flow(object):
+    def __init__(self, flow_type, value, time, time_type="int", currency="mxn"):
```

Comment:
Add docstring specifing the meaning of the object properties, and the values that they can take